src/context.c

```diff
@@ -571,6 +571,51 @@ lysc_node_clear_priv_dfs_cb(struct lysc_node *node, void *UNUSED(data), ly_bool
     return LY_SUCCESS;
 }
 
+static void
+lysc_node_clear_all_priv(struct lys_module *mod)
+{
+    LY_ARRAY_COUNT_TYPE u, v;
+    /* lys_compile_extension_instance() was called */
+    ly_bool lcei_called = 0;
```

Comment:
The comment is fine but the flag name does not need to be that specific, I would call it simply `ext_has_compiled_nodes` or something similar.